app/code/community/Bolt/Boltpay/Model/Admin/ExtraConfig.php

```diff
@@ -193,6 +228,7 @@ public function filterDatadogKeySeverity($rawConfigValue, $additionalParams = ar
     {
         $allExtraConfigs = (array)json_decode($this->normalizeJSON(Mage::getStoreConfig('payment/boltpay/extra_options')), true);
         return (array_key_exists("datadogKeySeverity", $allExtraConfigs)) ? $rawConfigValue : Bolt_Boltpay_Helper_DataDogTrait::$defaultSeverityConfig;
+        // return strlen(trim($rawConfigValue)) ? trim($rawConfigValue) : Bolt_Boltpay_Helper_DataDogTrait::$defaultSeverityConfig; # preferable way to get default
```

Comment:
no commented out code please

Reply:
This is here for future reference, but, I'll update the code.